scripts/prerm

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+rm -f ${CMAKE_INSTALL_FULL_LIBDIR}/python3/dist-packages/TestHarness
+rm -f ${CMAKE_INSTALL_FULL_DATAROOTDIR}/leap_testing/bin
```

Comment:
Sorry, I should have caught this before.
```suggestion
rm -rf ${CMAKE_INSTALL_FULL_LIBDIR}/python3/dist-packages
rm -rf ${CMAKE_INSTALL_FULL_DATAROOTDIR}/leap_testing
```

Reply:
I'm nervous about this change.  What if it is installed into a common location where `python3/dist-packages/` could have non-leap owner resources we don't want to just destroy.  I almost think I need to check at each level and only remove if the dir is empty.

Reply:
Good point about dist packages. You could do a combination  'rmdir -p' for the directors and rm -r for the symlinks, and that would protect us, I think.

Reply:
if you want to make it as simple as possible, I'd suggest removing the `mkdir`s since afaict they're guaranteed to be there by the time the `postinst` script runs. Then you're only responsible for removing the symlinks in `prerm` (unchanged from how it is now).

That allows the `postinst` & `prerm` to be 100% the opposite of each other without worrying about removing directories and whatnot.

Reply:
Here is the solution I implemented and tested: [c46400c](https://github.com/AntelopeIO/leap/pull/818/commits/c46400c527ea956c06217a8474f0541de6734447)

```
Best effort attempt to remove the directories we may have created during the install.

Can ignore failed directory deletion as they may be installing to location that has more than leap in it, for instance the ~/lib/python3/dist-packages, however if it is empty it is a safe assumption that it is in the location the install created it and can be removed as the inverse uninstall operation.
```